CSS Module landing page: filter effects (#24261)
mdn/content: files/en-us/web/css/filter_effects/index.md

  - Guide
  - Overview
  - Reference
spec-urls: 

>   - https://drafts.fxtf.org/filter-effects

Author: ```suggestion
  - https://drafts.fxtf.org/filter-effects-2/
```

  - Guide
  - Overview
  - Reference
spec-urls: 
  - https://drafts.fxtf.org/filter-effects-2

>   - https://drafts.fxtf.org/filter-effects-1

Reviewer: these links are showing as "Unknown specification" in the table

Reviewer 2: There should be only one, and I think it should be: https://drafts.fxtf.org/filter-effects/

Author: https://drafts.fxtf.org/filter-effects/ doesn't have backdrop-filter, neither does -1. And -2 doesn't have filter functions, so we need both here.

>   - https://drafts.fxtf.org/filter-effects-2
>   - https://drafts.fxtf.org/filter-effects-1

Author: ```suggestion
  - https://drafts.fxtf.org/filter-effects
```

---

{{CSSRef}}

> The **CSS filter effects** module defines a way of processing an element's rendering before it is displayed in the document.

Reviewer: ```suggestion
The properties in the **CSS filter effects** module let you define a way of processing an element's rendering before the element is displayed in the document. Examples of such effects include blurring and changing the intensity of the color of an element.
```

### Filter effects in action

> Play with the various sliders to apply filter effects to the image below:

Reviewer: I am not seeing any changes in the image when I move the sliders (chrome, firefox)

Author: https://github.com/mdn/css-examples/pull/116 fixes it. fingers crossed

{{EmbedGHLiveSample("css-examples/modules/filters.html", '100%', 420)}}

To see the code for this filter effects sample, [view the source on Github](https://github.com/mdn/content/blob/main/files/en-us/web/css/css_filter_effects/index.md).

### Properties

- {{cssxref("backdrop-filter")}}
- {{cssxref("filter")}}

### Functions

- {{cssxref("filter-function/blur", "blur()")}}
- {{cssxref("filter-function/brightness", "brightness()")}}
- {{cssxref("filter-function/contrast", "contrast()")}}
- {{cssxref("filter-function/drop-shadow", "drop-shadow()")}}
- {{cssxref("filter-function/grayscale", "grayscale()")}}
- {{cssxref("filter-function/hue-rotate", "hue-rotate()")}}
- {{cssxref("filter-function/invert", "invert()")}}
- {{cssxref("filter-function/opacity", "opacity()")}}
- {{cssxref("filter-function/saturate", "saturate()")}}
- {{cssxref("filter-function/sepia", "sepia()")}}

## Related concepts

- {{CSSxRef("&lt;image&gt;")}} data type
- {{cssxref("&lt;filter-function&gt;")}} data type

- {{cssxref("background-image")}} CSS property
- {{cssxref("background-blend-mode")}} CSS property
- {{cssxref("mix-blend-mode")}} CSS property

- {{glossary("interpolation")}} glossary term

> - SVG {{cssxref("color-interpolation-filters")}} property

Reviewer: how about 

```suggestion
- {{cssxref("color-interpolation-filters")}} SVG property
```
